lib/videojs-playlist-ui.js

```diff
@@ -81,14 +95,40 @@ class PlaylistMenuItem extends Component {
 
     Component.call(this, player, playlistItem);
 
+    this.playOnSelect = settings.playOnSelect;
+
     this.emitTapEvents();
 
-    this.on(['click', 'tap'], (event) => {
-      player.playlist.currentItem(indexOf(player.playlist(), this.item));
-      if (settings.playOnSelect) {
-        player.play();
-      }
-    });
+    this.on(['click', 'tap'], this.switchPlaylistItem_);
+    this.on('keydown', this.handleKeyDown_);
+    this.on(['focus', 'mouseover'], this.addPlayIcon_);
+    this.on(['blur', 'mouseout'], this.removePlayIcon_);
+
+  }
+
+  handleKeyDown_(event) {
+    if (event.which === 13 || event.which === 32) {
```

Comment:
oops forgot to comment about it.
videojs has an issue to switch to use a keycode library, we probably should expose it for plugins to use.